.github/workflows/crawler.yml

```diff
@@ -0,0 +1,23 @@
+# Auto-generated by Cimas: Do not edit it manually!
+# See https://github.com/metanorma/cimas
+name: Crawler
+
+on:
+  push:
+    branches: [ master, main ]
+    tags: [ v* ]
+  pull_request:
+  schedule:
+  - cron: '0 14 * * *'
+  workflow_dispatch:
+    inputs:
+      args:
+        description: Extra args to pass to crawler.rb script
+        required: false
+        default: 'latest'
```

Comment:
@CAMOBAP please add choice here

Reply:
To be on the same page the `crawler.rb` script don't process any args yet. They have just because 'mass-update' use a single template form `relaton/support` repo

I can remove inputs completely

Reply:
Question: if a GHA workflow is defined in repo X, and crawler.rb is in the same repo X, what is the use case for passing arguments to crawler.rb via GHA workflow if you could just modify crawler.rb accordingly?

Reply:
repo X reuse shared workflow from `relaton/support`. Due to security reasons shared workflow cannot access env/security vars from caller workflow

Reply:
@CAMOBAP maybe I am not familiar enough with this, but I am looking at this file: https://github.com/relaton/relaton-data-iana/blob/8926db9872d8ed851702a01f1989cd73c391e4d9/.github/workflows/crawler.yml

This file is in _this_ repository, not in relaton/support. And it defines arguments to be passed to crawler.rb. Which is also in this same repository:)

Reply:
I can see that the goal of this is to allow different arguments for each run specified when a human is triggering the workflow by hand. I guess that makes sense in some cases (like debug logging?).

Reply:
The crawler.yml from relaton-data-iana just copy of common file from support repository. We have a special tool called `cimas` which perform mass update for ci configuration once it changed. This is why we have top comment about this.

Regarding the point that crawler.rb and crawler.yml in the same repo, again, crawler.yml is just reuse crawler.yml from support, and declare ways how it can be triggered.

For some relaton-data-* repos crawler.rb need access to user input or secrets, but it can't because it was called from "shared" workflow from support repository